src/components/SignUpForm/SignUpForm.jsx

```diff
@@ -83,9 +83,11 @@ const SignUpForm = () => {
         localStorage.setItem("userProfile", JSON.stringify(userProfile));
         setUser(userProfile);
       }
-    } catch (response) {
-      setError({ message: "Something went wrong during sign up" });
-      console.error(response);
+    } catch (error) {
+      setError({
+        message:
+          error.response?.data.message || "Something went wrong during sign up",
```

Comment:
The reason I don't want to display the error message is because the error messages returned from aren't user friendly. To address this we can make sure that the response from the API only returns user friendly messages